crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -678,53 +682,111 @@ impl TryFrom<TransactionEffects> for SuiTransactionBlockEffects {
     }
 }
 
+fn owned_objref_string(obj: &OwnedObjectRef) -> String {
+    format!(
+        " ┌──\n │ ID: {} \n │ Owner: {} \n │ Version: {:?} \n │ Digest: {}\n └──",
```

Comment:
nit: it doesn't make a difference, but you don't want the debug format anymore for Version.
```suggestion
        " ┌──\n │ ID: {} \n │ Owner: {} \n │ Version: {} \n │ Digest: {}\n └──",
```